src/app/+item-page/simple/item-types/shared/item.component.ts

```diff
@@ -82,18 +87,53 @@ export const relationsToItems = (thisId: string, ids: ItemDataService) =>
       distinctUntilChanged(compareArraysUsingIds()),
     );
 
+/**
+ * Operator for turning a list of relationships into a list of metadatarepresentations given the original metadata
+ * @param thisId      The id of the parent item
+ * @param itemType    The type of relation this list resembles (for creating representations)
+ * @param metadata    The list of original Metadatum objects
+ * @param ids         The ItemDataService to use for fetching Items from the Rest API
+ */
+export const relationsToRepresentations = (thisId: string, itemType: string, metadata: Metadatum[], ids: ItemDataService) =>
+  (source: Observable<Relationship[]>): Observable<MetadataRepresentation[]> =>
+    source.pipe(
+      flatMap((rels: Relationship[]) =>
+        observableZip(
+          ...metadata.map((metadatum: Metadatum) => {
+            const prefix = 'virtual::';
```

Comment:
Isn't this metadata specific behavior, or shouldn't be this at metadata level?

Reply:
It should. I'll move it

Reply:
👍